src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -71,11 +71,10 @@ export const RETENTION_TIME_UNITS = [
 
 export interface TypeTestedSpec {
   is_derived: boolean;
-  specimen_type: Code | null;
   preference: Preference;
   container: ContainerSpec | null;
   requirement: string | null;
```

Comment:
```suggestion
  requirement?: string | null;
```

<img width="515" alt="image" src="https://github.com/user-attachments/assets/8a9214fe-b41c-448c-814c-bb6e15d31cbe" />